packages/map/src/map.component.ts

```diff
@@ -228,7 +229,8 @@ export class MapComponent extends RootComponent implements OnInit, AfterViewInit
 
     /**
      * With currently input this.markers, create markers on the Map
-     * @private
+     * FIXME: For some reason the @private doesn't pass linting, so it's commented out
```

Comment:
@private is unneeded for TS just as is adding the variable Type to @param, so it complains. Just having `private` in the function assignment should do it.

Reply:
Btw, didn't realize this was from me. PHPstorm loves to auto add those. It's been removed since in https://github.com/Sitetheory/stratus/pull/465